utility/improve-punctuation.js

```diff
@@ -0,0 +1,19 @@
+module.exports = improvePunctuation
```

Comment:
@anseljh: This module does the dirty work of "educating" quotes. As you'll see, it's a far cry from our previous efforts.